bjosefow_21_mvc_thymeleaf/src/main/resources/templates/home.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th = "http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Strona Glowna</title>
+    <style>
+        a {
+            display: block;
+        }
+    </style>
+</head>
+<body>
+<h1>Strona glowna</h1>
+<a href="/lista">Wyswietl wszytskie produkty</a>
+<div th:remove="all-but-first">
+    <a th:remove-but-first th:each="category: ${categories}" href="#" th:href="@{|/lista?category=${category.getDescription()}|}" th:text="|Wyswietl produkty o kategorii: ${category.getDescription()}|">Wyswietl produkty o kategorii: Art_spozywcze</a>
```

Comment:
Ten atrybut th:remove-but-first chyba nie jest potrzebny, a przynajmniej ja się z nim nigdy nie spotkałem, więc jeżeli ma jakieś uzasadnienie, to daj znać w komentarzu

Reply:
To byl tylko blad, docelowo chcialam uzyc tego co w div powyzej, ale zapomnialam usunac z elementu a.